examples/hosts/app-integration/external-controller/README.md

```diff
@@ -42,3 +42,26 @@ For in browser testing update `./jest-puppeteer.config.js` to:
 Dice Roller uses the following distributed data structures:
 
 - SharedDirectory - root
+
+## Backed Locally and running with live FRS instance
+
+We can connect to both live FRS instance by passing in the tenanID, orderer and storage as well as using the tenantID as "local" for running against Tinylicious for development purpose.
+
+For running the Tinylicious client, we pass the tenantID as "local" and make use of `InsecureTokenProvider` where we can pass anything into the key since we are running it locally and an object identifying the current user. For running the instance locally, it would get naviagted to the Tinylicious on the default values of `localhost:7070`.
```

Comment:
```suggestion
To run the the `FrsClient` against our local tinylicious instance, we pass the `tenantId` as "local" and make use of `InsecureTokenProvider`. For the latter, we pass in two values to its constructor: a key string, which can be anything since we are running it locally, and an object identifying the current user. For running the instance locally, the orderer and storage URLs would point to the Tinylicious instance on the default values of `http://localhost:7070`.
```